docs/index.rst

```diff
@@ -3,6 +3,18 @@
 Installing using Docker
 =======================
 
+.. toctree::
+    :maxdepth: 2
+    :hidden:
+
+    quickstart
+    security/index
```

Comment:
is this right? `docs/security/security.rst` moved to `docs/security.rst`, so shouldn't this just be `security`?

Reply:
sorry, you were reviewing the WIP